components/apimgt/org.wso2.carbon.apimgt.core/src/test/java/org/wso2/carbon/apimgt/core/SampleTestObjectCreator.java

```diff
@@ -466,6 +474,11 @@ public static Application createApplicationWithPermissions() {
         return application;
     }
 
+    /**
+     * create default api policy
+     *
+     * @return
```

Comment:
Shall we add the return object description. Check the whole PR. There are multiple places to do so